hicommonwealth/commonwealth
# Remove app.proposalEmitter.

## Link to Issue
Closes: #8364

## Description of Changes
- Uses a setter in the view proposal page to replace the global proposalEmitter.

## Test Plan
- Go to csdk community (or other cosmos community where you can create/vote on a proposal = own token), log in with an address that has csdk token.
- Create a proposal.
- Vote on the proposal. Verify that screen redraws to reflect vote once tx completes.
